rites/roman1969/__tests__/brazil-calendar.test.ts

```diff
@@ -11,83 +11,228 @@ describe('Testing Brazilian calendar specific celebrations', () => {
   let calendar2024: Awaited<ReturnType<typeof romcal.generateCalendar>>;
   let calendar2025: Awaited<ReturnType<typeof romcal.generateCalendar>>;
   let calendar2026: Awaited<ReturnType<typeof romcal.generateCalendar>>;
+  let calendar2027: Awaited<ReturnType<typeof romcal.generateCalendar>>;
 
   beforeAll(async () => {
     calendar2023 = await romcal.generateCalendar(2023);
     calendar2024 = await romcal.generateCalendar(2024);
     calendar2025 = await romcal.generateCalendar(2025);
     calendar2026 = await romcal.generateCalendar(2026);
+    calendar2027 = await romcal.generateCalendar(2027);
   });
 
-  describe('Brazilian saints', () => {
+  describe('Brazilian celebrations', () => {
```

Comment:
you should move other celebrations to its own `describe` and maintain saints where it was already defined